packages/interface/src/components/options-menu/README.md

```diff
@@ -0,0 +1,71 @@
+# OptionsMenu
```

Comment:
I'm not sold of the new name prefixed with `OptionsMenu`. We have existing SlotFills for the extenders that start with `MoreMenu`:

https://github.com/WordPress/gutenberg/tree/trunk/packages/edit-post#pluginmoremenuitem
https://github.com/WordPress/gutenberg/tree/trunk/packages/edit-post#pluginsidebarmoremenuitem

Internally they use:
https://github.com/WordPress/gutenberg/tree/trunk/packages/interface/src/components/complementary-area-more-menu-item
https://github.com/WordPress/gutenberg/tree/trunk/packages/interface/src/components/action-item

It would be great to consolidate this code and align naming conventions.

Reply:
Oh, that's a very good point. I'll change it back!

Reply:
Done 👍